cinema/views.py

```diff
@@ -127,6 +118,29 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type=OpenApiTypes.STR,
+                description="Filter movies by title",
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter movies by genre",
+            ),
+            OpenApiParameter(
+                name="actors",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter movies by actors",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        """ Get a list of all available buses """
```

Comment:
The docstring for the `list` method in `MovieViewSet` mentions 'buses' instead of 'movies'. This seems to be a copy-paste error and should be corrected to accurately describe the functionality.